AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/Mocks/MockAuthPasswordlessBehavior.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+@testable import AWSPluginsCore
+@testable import AWSCognitoAuthPlugin
+
+class MockAuthPasswordlessBehavior: AuthPasswordlessBehavior {
+    
+    public var preInitiateAuthSignUpCallCount = 0
```

Comment:
it will be accessed in from these file paths `~/TaskTests/ClientBehaviorTests` and mock is in a folder level same as `TaskTests` when sign up calls are added